src/ontology/edits/oeo-physical.omn

```diff
@@ -1160,6 +1168,7 @@ Class: OEO_00000021
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A hard coal power unit is a coal power unit using hard coal as fuel.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "Steinkohlekraftwerkblock"@de,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000118> "Steinkohlekraftwerksblock"@de,
```

Reply:
According to the [Google Ngram Viewer](https://books.google.com/ngrams/graph?content=Kraftwerkblock%2CKraftwerksblock&year_start=1800&year_end=2019&corpus=de-2019&smoothing=3) the spelling with [Fugen-S](https://de.wikipedia.org/wiki/Fugen-S) is much more common.